File: task3/helloworld.cpp
#include <iostream>
#include <cmath>

bool isPrime(int num) {
    if (num < 2) {
        return false;
    }
    for (int i = 2; i <= sqrt(num); i++) {
        if (num % i == 0) {
            return false;
        }
    }
    return true;
}

int main() {
    int n;
    std::cout << "Enter a number: ";
    std::cin >> n;

    std::cout << "Prime numbers up to " << n << ":\n";
    for (int i = 2; i <= n; i++) {
        if (isPrime(i)) {
            std::cout << i << std::endl;
        }
    }

    return 0;
}

File: task3/helloworld.rb
def is_prime(num)
    return false if num < 2
  
    (2..Math.sqrt(num).to_i).each do |i|
      return false if num % i == 0
    end
  
    true
  end
  
  print "Enter a number: "
  n = gets.chomp.to_i
  
  puts "Prime numbers up to #{n}:"
  (2..n).each do |i|
    puts i if is_prime(i)
  end
  